cerberus_anymal_control_common/cerberus_anymal_control_base/include/cerberus_anymal_control_base/CerberusAnymalControlBase.tpp

```diff
@@ -0,0 +1,70 @@
+/*!
+ * @name      CerberusAnymalControlBase.tpp
+ * @author    Samuel Zimmermann
+ * @email     zsamuel@ethz.ch
+ * @date      April 30, 2020
+ *
+ * Copyright (C) 2020 Robotic Systems Lab, ETH Zurich.
+ * All rights reserved.
+ * http://www.rsl.ethz.ch/
+ */
+
+#include "CerberusAnymalControlBase.hpp"
+
+namespace cerberus_anymal_control {
+
+template <typename ConcreteQuadrupedController>
+CerberusAnymalControlBase<ConcreteQuadrupedController>::CerberusAnymalControlBase(int numberOfJoints) :
+    quadrupedController_(),
+    graph_(),
+    numberOfJoints_(numberOfJoints),
+    genCoordinates_(),
+    genVelocities_(),
+    desLinearVelocity_(Eigen::Vector3d::Zero()),
+    desAngularVelocity_(Eigen::Vector3d::Zero()),
+    desJointPositions_(Eigen::VectorXd::Zero(numberOfJoints)),
+    hasBodyVelocityData_(false),
+    hasJointPositionData_(false),
+    initializedGraph_(false)
+{
+  quadrupedController_ = std::make_unique<ConcreteQuadrupedController>();
+  if (numberOfJoints_ != 12) {
+    std::cout << "[CerberusAnymalControlBase::CerberusAnymalControlBase] Joint Number is not equal to 12! Controller might not be working properly." << std::endl;
```

Comment:
why do you use std::cout instead of ROS_INFO? Also, if you like ou can use MELO ([open source](https://bitbucket.org/leggedrobotics/message_logger/src/master/)) and add it as submodule to this repo. Comment applies to wherever you use std::cout in this PR

Reply:
I agree with ROS_INFO, but I disagree with MELO. I would keep the dependencies on extra packages to a minimum.

Reply:
The point is that this class has no ROS dependency at the moment. So I would have to add the ROS header just for printing.

Reply:
That's a valid point, but in my opinion it wouldn't make a big difference, given that our package is dependent on `roscpp` anyway.  
You could also go with a minimal include, like `#include <ros/console.h>` or so.